db/users.js

```diff
@@ -0,0 +1,93 @@
+const { client } =require("./client");
+const bcrypt = require('bcrypt');
+
+async function createUser({ username, password }) {
+    const SALT_COUNT = 10;  
+    const hashedPassword = await bcrypt.hash(password, SALT_COUNT);
+
+    try {
+        const { rows: [user], } = await client.query(
+            `
+            INSERT INTO users( username, password )
+            VALUES ($1, $2)
+            ON CONFLICT (username) DO NOTHING
+            RETURNING *;
+            `, [username, hashedPassword]
+        ); 
+        delete user.password;
+        return user;
+
+    } catch (err) {
+        throw Error(`Error while creating user: ${err}`)
+    }
+}
+
+async function getUser({username, password }) {
+    try {
+        const { rows: [user], } = await client.query(`
+        SELECT * FROM users WHERE username=$1
+        `, [username]);
+
+        const hashedPassword = user.password 
+        const passwordsMatch = await bcrypt.compare(password, hashedPassword);
+
+        if(passwordsMatch) {
+            delete hashedPassword
```

Comment:
Are these changes from Sean's code? There were a few issues like this one (should be "delete user.password") that I already reviewed for him